src/AstSemanticChecker.cpp

```diff
@@ -433,6 +433,19 @@ void AstSemanticChecker::checkLiteral(
 
 void AstSemanticChecker::checkAggregator(
         ErrorReport& report, const AstProgram& program, const AstAggregator& aggregator) {
+
+    auto* inner = nullptr;
+
+    visitDepthFirst(aggregator, [&](const AstAggregator& innerAgg) {
+      	if (aggregator != innerAgg) {
```

Comment:
There are still indentation problems here, see travis for more detail: https://travis-ci.org/souffle-lang/souffle/jobs/553063235

Reply:
how do i see more detail on travis? i am looking at it but i am not sure what i am looking for on it exactly

Reply:
Oh okay I noticed the output.